<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGALILEIWin</class>
 <widget class="QMainWindow" name="QGALILEIWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qgalileiwin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="Desktop"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="aConnect"/>
    <addaction name="aDisconnect"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menuKnowledge">
    <property name="title">
     <string>Knowledge</string>
    </property>
    <addaction name="aDicts"/>
   </widget>
   <widget class="QMenu" name="menuDocuments">
    <property name="title">
     <string>Documents</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTopics">
    <property name="title">
     <string>Topics</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCommunities">
    <property name="title">
     <string>Communities</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="aCloseAll"/>
    <addaction name="separator"/>
    <addaction name="aTile"/>
    <addaction name="aCascade"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="aConfigSession"/>
    <addaction name="separator"/>
    <addaction name="aConfigQGALILEI"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuKnowledge"/>
   <addaction name="menuDocuments"/>
   <addaction name="menuTopics"/>
   <addaction name="menuUsers"/>
   <addaction name="menuCommunities"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuWindow"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="aExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="aConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="aDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="aDicts">
   <property name="text">
    <string>&amp;Dictionnaries</string>
   </property>
  </action>
  <action name="aCloseAll">
   <property name="text">
    <string>&amp;Close All</string>
   </property>
  </action>
  <action name="aTile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="aCascade">
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="aConfigSession">
   <property name="text">
    <string>Configure &amp;Session</string>
   </property>
  </action>
  <action name="aConfigQGALILEI">
   <property name="text">
    <string>Configure &amp;QGALILEI</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
